### ISSUE TITLE
Security vulnerability linked to nodemon

### ISSUE THREAD

**Issue by p-j** [bee-queue/arena#136]
Hi,

A security vulnerability was found in event-stream  https://github.com/dominictarr/event-stream/issues/116
Any version past 3.3.4 may be vulnerable.

A vulnerable version of EventStream is present in the dependency tree of bull-arena via nodemone@1.18.4
```
└─┬ bull-arena@2.5.0
  └─┬ nodemon@1.18.4
    └─┬ pstree.remy@1.1.0
      └─┬ ps-tree@1.1.0
        └─┬ event-stream@3.3.6 
          └── flatmap-stream@0.1.2
```

It is advised to upgrade to `nodemon@^1.18.7` to get rid of the problematic dependency tree.

**Comment by p-j** [comment]
Linked to #137

**Comment by skeggse** [comment]
Thanks for flagging. This isn't presently a problem because the lockfile [pins event-stream to 3.3.4](https://github.com/bee-queue/arena/blob/e48aa7391a9e8cf777dd0032b8b0a3f508d228fb/package-lock.json#L1079). I'll update `nodemon` regardless.

**Comment by eeVoskos** [comment]
When should we expect a new release on npm @skeggse? Cloud services are already "protesting" against our builds with the old `nodemon` version! :-/

**Comment by p-j** [comment]
> Thanks for flagging. This isn't presently a problem because the lockfile [pins event-stream to 3.3.4](https://github.com/bee-queue/arena/blob/e48aa7391a9e8cf777dd0032b8b0a3f508d228fb/package-lock.json#L1079). I'll update `nodemon` regardless.

It isn't for a straight use of arena, but in our case we use it programatically inside an existing express server and as we use `yarn` as a package manager, the package-lock.json isn't followed.
The description of the dependency tree above is actually from one of my app.
It's not a problem per se because getting rid of the node_modules and reinstalling grabs the newest version of nodemon, but pushing a patch version of bull-arena would prompt people that maybe unaware of the vulnerability to update to a safer version.

**Comment by eeVoskos** [comment]
There is a problem specifically with cloud deployments (namely Google AppEngine). Even if you replace the dependency in package-lock.json, `npm i` is run by the build system and replaces `nodemon` with the "malicious" version, causing build errors.

**Comment by skeggse** [comment]
Published as `v2.5.1`.
